lib/Dialect/TritonGPU/IR/Dialect.cpp

```diff
@@ -559,8 +559,8 @@ SmallVector<unsigned> getCTASplitNum(Attribute layout) {
                mmaLayout.getCTALayout().getCTASplitNum().end());
 #ifdef USE_ROCM
   } else if (auto mfmaLayout = layout.dyn_cast<MfmaEncodingAttr>()) {
```

Comment:
also could be removed

Reply:
Thank you!
Let's fix it in some following PRs